### PR TITLE
Bug 1142491 -  Provide a page with third-party library attribution

### PR DESCRIPTION
**Pull request by alessiobrozzi1**

### CODE REVIEW

File: Client.xcodeproj/xcshareddata/xcschemes/Account.xcscheme [hunk 1]
```diff
@@ -43,6 +43,16 @@
                </Test>
             </SkippedTests>
          </TestableReference>
+         <TestableReference
```
Comment (fluffyemily) [inline Client.xcodeproj/xcshareddata/xcschemes/Account.xcscheme]: These are not needed, can these changes please be removed.

Comment (fluffyemily) [inline Client.xcodeproj/xcshareddata/xcschemes/Account.xcscheme]: And for all following

File: Client/Assets/About/thirdyparty.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<h1>Third-Party Attribution</h1>
+
+<h2>Firefox for iOS</h2>
+<p class="link"><a href="https://github.com/mozilla/firefox-ios">github.com/mozilla/firefox-ios</a></p>
+<div class="text">
+
+
+
+
+<h3 id="mozilla-public-license-version-2.0">Thirdy Party<br>Version 2.0</h3>
```
Comment (fluffyemily) [inline Client/Assets/About/thirdyparty.html]: `Third` Party

**Comment by alessiobrozzi1** [comment]
I implemented the requested changes. I looked up on how to add a feature flag around my changes and tried following [this guide](https://github.com/mozilla-mobile/firefox-ios/wiki/Adding-a-Feature-Flag), however I can't seem to locate the specified file FeatureFlags.xcconfig in the project. Should I follow another guide?

Thanks

**Comment by farhanpatel** [comment]
We already have this in a similar form under the Licenses which you can find in settings. It first contains the License for Firefox itself and then for any third parties we use. It also contains links to their github pages.

**Comment by alessiobrozzi1** [comment]
Hi Emily, just wondering if I should contact the Mozilla UX team to take a look at thirdparty.html... Moreover, I added a sample paragraph to thirdparty.html of third-party attribution for one of the libraries Firefox uses.

**Comment by farhanpatel** [comment]
Here is the existing License page 
https://github.com/mozilla-mobile/firefox-ios/blob/master/Client/Assets/About/Licenses.html

**Comment by farhanpatel** [comment]
Hey Alessio, 
I've confirmed with Emily this is something we already have and don't need a separate page for it. We do have some libraries that might be missing from that page. You are welcome to create a Bug and file a PR for those missing libraries! 

I (and Emily) apologize that you spent time on this feature and that we wont be able to use it 😢

**Comment by alessiobrozzi1** [comment]
Hi Farhan,
It’s true that the License page contains third-party attribution, however it’s good practice to have a separate page for third-party attributions since they’re more accessible this way and this is how the most popular apps on the App Store do (Google Chrome/Maps, Amazon, Youtube, Spotify, Facebook etc.. See attached).
This is just some info in case you might consider reopening the bug.
amazon
facebook
google_chrome
google_maps
spotify
youtube

**Pull request closed without merging**